README.md

```diff
@@ -1,63 +1,351 @@
-# Unikraft "hello world" Application
+# A Simple "Hello, world!" Application on Unikraft
```

Comment:
```suggestion
# A "Hello, world!" Application on Unikraft
```

Avoid using simple, simply, easy, easily. It's never easy for newcomers, and it gives the impression that they're not up to the job.